openlibrary/templates/search/advancedsearch.html

```diff
@@ -3,8 +3,7 @@ <h1>$_('Advanced Search')</h1>
   <form method="get" action="/search" class="siteSearch">
     <fieldset class="home">
       <legend>$_('Advanced Search')</legend>
-      <label for="qtop" class="hidden">$_('Keyword')</label>
-      <input type="hidden" name="q" id="qtop" size="35">
+      <input name="q" id="qtop" size="35" style="display: none;">
```

Comment:
Seems like `type=hidden` is preferable, according to https://stackoverflow.com/a/13422440/2317712 . So we can just remove the `size` attribute, since it only affects appearance

```suggestion
      <input type="hidden" name="q" id="qtop">
```